Jenkinsfile

```diff
@@ -228,6 +235,21 @@ ENVIRONMENT_EOF
       '''
   }
 
+  if (env.CASSANDRA_VERSION.split('-')[0] == 'hcd') {
```

Comment:
Presumably this should be an "else if" in order to avoid some weird case in which both of these if blocks execute.  There's no obvious way for that to happen but in the worst case it still clearly communicates intent.